### PR TITLE
[To rel/0.12] Fix load with time partition enable

### PR DESCRIPTION
**Pull request by yschengzi**
IoTDBLoadExternalTsfileIT.loadUnsequenceTsfile fails on CI for unix(16, ubuntu-latest) sometimes, 
make this branch to find what leads this results.

### CODE REVIEW

**Review by wangchao316 — CHANGES_REQUESTED** [review]
Thanks for your contribution.
but this hava a little question.              
+ " "  + "subsequenceTime: "     ====       "       subsequenceTime:"

**Comment by yschengzi** [comment]
> Thanks for your contribution.
> but this hava a little question.
> 
> * " "  + "subsequenceTime: "     ====       "       subsequenceTime:"

i see, thanks for reminding

File: server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java [hunk 1]
```diff
@@ -191,13 +191,13 @@ private void deleteLevelFile(TsFileResource seqFile) {
       List<TsFileResource> result = new ArrayList<>();
       if (sequence) {
         List<SortedSet<TsFileResource>> sequenceTsFileList =
-            sequenceTsFileResources.get(timePartition);
+            sequenceTsFileResources.getOrDefault(timePartition, new ArrayList<>());
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java:194]: Should we use computeIfAbsent to set the ArrayList back?

Comment (yschengzi) [reply server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java:194]: ok, computeIfAbsent is a better choice

Comment (HTHou) [reply server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java]: It seem that the tests failed if change here…

Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java:194]: this only for a check, he will add the tsfile in another place

**Review by HTHou — APPROVED** [review]

**Pull request merged**